config/vufind/config.ini

```diff
@@ -1868,6 +1879,18 @@ RIS = false
 ; export.ini for more details on available options.
 ;defaultType = download
 
+; This overrides the limit of exports in the BulkActions section per method. Note that they
+; can not exceed that limit and can only be used to further restrict the number of items that
+; can be used per export.
+limits["RefWorks"] = 100
```

Comment:
There might be practical reasons to leave this here, but did you consider putting these settings in Export.ini instead? It seems like it might be more desirable to put these configurations in a more specific place, to reduce noise in config.ini and to remind people to set this value when adding new export methods.

Reply:
Moved it. But just realized that the explanation is missing in the comment of export.ini. I will add it.